chaosazure/machine/actions.py

```diff
@@ -300,34 +280,17 @@ def fill_disk(filter: str = None,
     Fill two machines at random from the group 'rg'
     """
 
-    logger.debug(
-        "Start fill_disk: configuration='{}', filter='{}'".format(
-            configuration, filter))
+    msg = "Starting fill_disk: configuration='{}', filter='{}'," \
```

Comment:
I know this not a high performance application but CPython compiler is rather a 'conservative' compiler when we talk about optimization do in the end we will generate few more commands here.

General rule would be if we do not need additional variables we should not create one. I guess intention here was to create more readable code but on the other hand it may confuse somebody who read this code as one can expect that this `msg` is used in more places

Reply:
Yes. Readability was indeed my intention. Okay. Will remember it the next times and refactor here a bit

Reply:
It's ok to have it like that. I guess that here readability is more important for us than performance ;)